group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/CurrentAssignmentBuilder.java

```diff
@@ -33,49 +33,6 @@
  * The CurrentAssignmentBuilder class encapsulates the reconciliation engine of the
  * consumer group protocol. Given the current state of a member and a desired or target
  * assignment state, the state machine takes the necessary steps to converge them.
- *
- * The member state has the following properties:
- * - Current Epoch:
```

Comment:
Do we have a replacement summary for this java doc? I noticed there were more comments in the code, but not sure if they covered everything.

Reply:
Indeed. I moved the description closer to the code.